src/tabs/power.html

```diff
@@ -192,6 +192,12 @@
                 i18n="powerVoltageMultiplier"></span>
             </label>
         </div>
+        <div class="number">
+            <label> <input type="number" name="vbatcalibration" step="0.01" min="0" max="255" /> <span
+                i18n="powerVoltageCalibration"></span>
+                <div class="helpicon cf_tip" i18n_title="powerVoltageCalibrationHelp"></div>
```

Comment:
There is also a problem with the tooltips stopping to work when the user selects a source that hides them, then selects one that shows them again.